src/sentry/static/sentry/app/utils/retryableImport.jsx

```diff
@@ -7,7 +7,12 @@ export default async function retryableImport(fn) {
       const module = await fn();
       return module.default || module;
     } catch (err) {
-      if (retries < MAX_RETRIES) {
+      const isWebpackLoadingError =
+        err &&
+        typeof err.message === 'string' &&
+        err.message.toLowerCase().includes('loading chunk');
```

Comment:
We have two very similar blocks of logic is it worth having a utility function to check in an error is webpack chunk related?